dspace-server-webapp/src/test/java/org/dspace/app/rest/ItemRestRepositoryIT.java

```diff
@@ -3021,10 +3021,10 @@ public void testHiddenMetadataForUserWithWriteRights() throws Exception {
         String token = getAuthToken(eperson.getEmail(), password);
 
         getClient(token).perform(get("/api/core/items/" + item.getID()))
-                        .andExpect(status().isOk())
-                        .andExpect(jsonPath("$", ItemMatcher.matchItemProperties(item)))
-                        .andExpect(jsonPath("$.metadata", matchMetadata("dc.title", "Public item 1")))
-                        .andExpect(jsonPath("$.metadata", matchMetadataDoesNotExist("dc.description.provenance")));
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$", ItemMatcher.matchItemProperties(item)))
+            .andExpect(jsonPath("$.metadata", matchMetadata("dc.title", "Public item 1")))
+            .andExpect(jsonPath("$.metadata", matchMetadata("dc.description.provenance", "Provenance data")));
```

Comment:
Because this test needs to be modified, we no longer have a test here that verifies an authenticated user *without* proper permissions *cannot* see the hidden metadata.    I'd recommend adding a new test that verifies that an authenticated user with `READ` permissions on the Item **cannot** see the `dc.description.provenance` field.  (It's important for us to have a test that proves that some authenticated users cannot see this hidden metadata)